### PR TITLE
Detect the left palm and fist. 

### PR DESCRIPTION
**Pull request by codieorberson**
Ignore the frame contrast file. That is only going to be used if the Palm gesture can not be detected within five ft on a 720p webcam. This pull request is only to detect the left palm and fist.

### CODE REVIEW

File: guiManager.py [hunk 1]
```diff
@@ -61,6 +61,9 @@ def start(self, loop_callback, close_callback):
         self.blink_label = self.gui.get_blink_label()
         self.fist_label = self.gui.get_fist_label()
         self.palm_label = self.gui.get_palm_label()
+        
+        self.low_contrast = self.gui.get_low_contrast()
+        self.high_contrast = self.gui.get_high_contrast()
```
Comment (lgint8042) [inline guiManager.py]: These two lines throw errors on my machine. Are they supposed to be `set` instead of `get`?

Comment (lgint8042) [reply guiManager.py]: Ran it with them commented out and it ran just fine. This was the only issue I had.

**Review by Danielle333 — COMMENTED** [review]
Commenting out those two lines worked for me too.  On my computer the frame (red box) that goes around the gesture is displayed all the way off-center of the gesture and to the right of it. If I make a left fix the box appears by my thumb. It worked on detecting my left fist and palm.

File: guiManager.py (hunk 1 above)
Comment (codieorberson) [reply guiManager.py]: Those were there for the scale bars to change the contrast for the palm. Theyre not needed, so I will take them out

**Comment by codieorberson** [comment]
Danielle, Yes I realized that. I'll look into it. I think its because im also flipping the redbox to appear on the other side. Not sure

**Review by Danielle333 — APPROVED** [review]

File: nonVerbalSmartHomeRecognitionSystem.py [hunk 2]
```diff
@@ -113,35 +120,40 @@ def __set_up_commands__(self):
...
+        else:
+            self.cap.set(cv2.CAP_PROP_FRAME_WIDTH, 600)
+            self.cap.set(cv2.CAP_PROP_FRAME_HEIGHT, 400)
+            self.valid_webcam = False
+            return False
+        
     def __set_up_camera__(self): 
-        self.cap = cv2.VideoCapture(0)
-        self.cap.set(cv2.CAP_PROP_FRAME_WIDTH, 500)
+        self.cap = cv2.VideoCapture(0, cv2.CAP_DSHOW)
```
Comment (lgint8042) [inline nonVerbalSmartHomeRecognitionSystem.py]: Not sure if the cv2.CAP_DSHOW was added for, but it does cause a problem with actually opening the camera on my laptop. Not sure why it does, however it does run just fine when I remove it. Not sure how it reacts on others laptops, but I think it's probably just something with mine?

**Review by lgint8042 — APPROVED** [review]

**Comment by lgint8042** [comment]
Also, can you resolve the conflicts with master before you merge to make sure all your changes are in there correctly?

**Pull request merged**